spec/std/regex_spec.cr

```diff
@@ -114,6 +114,9 @@ describe "Regex" do
     end
 
     it "matches a large single line string" do
+      LibPCRE.config LibPCRE::CONFIG_JIT, out jit_enabled
+      pending! "PCRE JIT mode not available." unless 1 == jit_enabled
```

Comment:
_Other way around_ notation is not rly used in stdlib. Personally, I find it confusing too.

```suggestion
      pending! "PCRE JIT mode not available." unless jit_enabled == 1
```

Reply:
Yoda conditionals are better because you could easily type `jit_enabled = 1` and it compile and you have a bug. Doing it this way produces a compile time error since you cant re-assign a literal.

Reply:
Even though your argument is technically correct, with peer-reviewed and tested code that doesn't happen that _easily_ - it did not happen to me for yrs and I use just eyes for discerning one or two equal signs, same goes for many projects I've participated in throughout that time 🤷‍♂️  Another thing is that it adds cognitive load - now, you have read (negated) expression _backward_ to make sense out of it... And last but not least: (after wickedpedia) "Yoda conditions are part of the coding standards for Symfony and WordPress." - Seeing such strong support just adds to the understanding, especially the _WordPress_ 🤣  I'd advise reading the section on [Criticism](https://en.wikipedia.org/wiki/Yoda_conditions#Criticism) in which you'll find cognitive load mentioned.

Reply:
Either is fine. We won't establish a style guide in this PR.